### ISSUE TITLE
Installation fails in Django 1.8

### ISSUE THREAD

**Issue by BobBowles** [bitmazk/django-calendarium#32]
`Pip install django-calendarium` also installs `South`, which is imcompatible with Django >= 1.7. When you try to run the migrations they fail with a message about missing South database modules. If you uninstall South it fails because the South module is detected as missing....

**Comment by Tyrdall** [comment]
@BobBowles This shoud help: https://github.com/bitmazk/django-calendarium/commit/af3f0be6f18230389f25d6272800ba048a018511

**Comment by BobBowles** [comment]
Thanks for your prompt response. Your fix has solved the South problem. 

Initially I still struggled with the migration stage, but eventually I worked out Django was clever enough to work things out for itself!

Instead of:

    python manage.py migrate calendarium

I did:

    python manage.py migrate

The first one failed with a big traceback relating to contenttypes needing to be migrated first. Since I had no idea what that meant I tried the second one (no app specified) and Django pulled in everything it needed (including the contenttypes migrations and calendarium with all its dependencies) without further hassle.

I'm not sure how applicable this might be to other people, but it might be worth a small update to the installation instructions...

Regards   Bob Bowles
